### PR TITLE
#212Add_Table_ResourceRequestCV

### PR DESCRIPTION
**Pull request by tiendat094**
add new Table resourceRequestCV 
#212 

image

### CODE REVIEW

File: angular/src/app/modules/delivery-management/delivery/request-resource-tab/form-resource-requestCV/form-resource-requestCV.component.ts [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  //     cvName : this.resourceRequestCV.cvName,
+  //     cvPath : this.resourceRequestCV.cvPath,
+  //     status : this.resourceRequestCV.status,
+  //     note : this.resourceRequestCV.note,
+  //     kpiPoint : this.resourceRequestCV.kpiPoint,
+  //     interviewDate : this.resourceRequestCV.interviewDate,
+  //     sendCVDate : this.resourceRequestCV.sendCVDate,
+  //     resourceRequestId : this.resourceRequestCV.resourceRequestId,
+  //  }
+   if(this.input.command=="create"){
```
Comment (tiennguyenhuuncc) [inline angular/src/app/modules/delivery-management/delivery/request-resource-tab/form-resource-requestCV/form-resource-requestCV.component.ts:118]: remove console.log

File: aspnet-core/src/ProjectManagement.Application/APIs/ResourceRequests/ResourceRequestAppService.cs [hunk 2]
```diff
@@ -72,6 +72,128 @@ public ResourceRequestAppService(
             _uploadFileService = uploadFileService;
         }
 
+
+        [HttpPost]
+        public async Task<ResourceRequestCVDto> AddCV(ResourceRequestCVDto resourceRequestCV)
+        {
+            var check = await WorkScope.GetAll<ResourceRequestCV>().AnyAsync(s => (s.UserId == resourceRequestCV.UserId && s.ResourceRequestId == resourceRequestCV.ResourceRequestId));
```
Comment (chinhvuquang) [inline aspnet-core/src/ProjectManagement.Application/APIs/ResourceRequests/ResourceRequestAppService.cs]: bỏ dấu ()

File: aspnet-core/src/ProjectManagement.Application/APIs/ResourceRequests/ResourceRequestAppService.cs [hunk 3]
```diff
@@ -72,6 +72,128 @@ public ResourceRequestAppService(
...
+                       KpiPoint = s.KpiPoint,
+                       InterviewDate = s.InterviewDate,
+                       SendCVDate = s.SendCVDate,
+                       Id = s.Id,
+                   }).ToListAsync();
+        }
+        [HttpGet]
+        public async Task<ResourceRequestCV> GetResourceRequestCVById(long resourceRequestCVId)
+        {
+            var check = await WorkScope.GetAll<ResourceRequestCV>().AnyAsync(s => (s.Id == resourceRequestCVId));
```
Comment (chinhvuquang) [inline aspnet-core/src/ProjectManagement.Application/APIs/ResourceRequests/ResourceRequestAppService.cs]: bỏ dấu ()

File: aspnet-core/src/ProjectManagement.Application/APIs/ResourceRequests/ResourceRequestAppService.cs [hunk 4]
```diff
@@ -72,6 +72,128 @@ public ResourceRequestAppService(
...
+        {
+            var rCV = await WorkScope.GetAsync<Entities.ResourceRequestCV>(resourceRequestCV.Id);
+            var nRs = ObjectMapper.Map<ResourceRequestCVDto, ResourceRequestCV>(resourceRequestCV, rCV);
+            await WorkScope.UpdateAsync<Entities.ResourceRequestCV>(nRs);
+            return resourceRequestCV;
+        }
+        [HttpDelete]
+        public async Task DeleteResourceRequestCV(long resourceRequestCVId)
+        {
+            var check = await WorkScope.GetAll<ResourceRequestCV>().AnyAsync(s => (s.Id == resourceRequestCVId));
```
Comment (chinhvuquang) [inline aspnet-core/src/ProjectManagement.Application/APIs/ResourceRequests/ResourceRequestAppService.cs]: bỏ dấu ()

**Review by chinhvuquang — APPROVED** [review]

**Pull request merged**